store/src/sections/404/index.js

```diff
@@ -2,15 +2,21 @@ import React from 'react'
 import Link from 'next/link'
 import { getRoute } from '../../utils'
 
-export const PageNotFound = () => {
+export const PageNotFound = ({ config }) => {
+   console.log(config)
    return (
       <div className="hern-404">
+         <img
+            src={config.illustrationImage.value}
+            alt="404 Not Found Image"
+            style={{ maxWidth: '25vw' }}
+         />
```

Comment:
Should be rendered conditionally so that is any instance doesn't have the the config its works properly
```suggestion
        {config?.illustrationImage?.value && <img
            src={config?.illustrationImage?.value}
            alt="404 Not Found Image"
            style={{ maxWidth: '25vw' }}
         />}
```